### PR TITLE
Clarify use of Mendix SSO

### PR DESCRIPTION
**Pull request by MarkvanMents**
[TW-2052]

### CODE REVIEW

File: content/en/docs/appstore/modules/mendix-sso.md [hunk 1]
```diff
@@ -33,18 +37,18 @@ You can use this module when you want your application's end-users to sign in wi
 
 ### 1.3 Limitations
 
+{{% todo %}}Are we still working on improving the volume throughput issues?{{% /todo %}}
+
+* Mendix SSO is not recommended for Production deployments as the Mendix SSO sign in screens are Mendix branded.
 * Due to the large number of app creations, synchronizing newly created apps might take some time. When an app is not yet synchronized, the deployment will not work when you attempt to deploy your app to the Mendix Developer Portal, as it will not recognize your app. In these cases, please be patient and try again later or contact [Mendix Support](https://support.mendix.com/) if the problem persists. Mendix is working on resolving the volume throughput issues.
```
Comment (MarkvanMents) [inline content/en/docs/appstore/modules/mendix-sso.md]: Are we still working on making Mendix SSO scalable, or should I remove this?

**Review by MarkvanMents — COMMENTED** [review]
I've made changes to the three documents initially discussed in the Jira issue [TW-2052].
There is other discussion of using Mendix SSO  - does any of this need to be changed? - I'll add the discussion to the Jira issue.

File: content/en/docs/appstore/modules/mendix-sso.md (hunk 1 above)
Comment (JaapF) [reply content/en/docs/appstore/modules/mendix-sso.md]: Let's not formulate this as a limitation for Mendix SSO, but instead give a more guiding description in the 'Typical use case' section. I'll send you a suggestion tomorrow.

File: content/en/docs/appstore/modules/mendix-sso.md [hunk 2]
```diff
@@ -33,18 +37,16 @@ You can use this module when you want your application's end-users to sign in wi
...
 * This module does not work for [native mobile](/refguide/native-mobile/) apps.
-* The default app `Logout` action resolves to the origin location found in a session cookie, which (re)triggers the `/openid/login/` endpoint, which logs the end-user in again. 
+* The default app `Logout` action resolves to the origin location found in a session cookie, which (re)triggers the `/openid/login/` endpoint, which logs the end-user in again.
 
 ### 1.4 Dependencies
 
-Your app has to be deployed on the Mendix Cloud in order to use this module.
+Your app has to be deployed on the Mendix Cloud in order to use this module. 
+Mendix Single Sign-On is only activated when your app is deployed to the Mendix Cloud. When you run your app locally, or on another cloud, you will need to use local credentials.
```
Comment (JaapF) [inline content/en/docs/appstore/modules/mendix-sso.md]: Let's mention all alternatives:

"When you....cloud, you can use either local credentials and use "Forgot my Password" module, or achieve SSO via OOIDc SSO module or SAML module."

File: content/en/docs/appstore/modules/mendix-sso.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/mendix-sso.md]: Removed: https://github.com/mendix/docs/pull/5771/commits/30232ffaa80382114d0508a2a9fb5eed9cddf5c4

File: content/en/docs/appstore/modules/mendix-sso.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/mendix-sso.md]: Done: https://github.com/mendix/docs/pull/5771/commits/30232ffaa80382114d0508a2a9fb5eed9cddf5c4

**Review by MarkvanMents — COMMENTED** [review]
@JaapF 
I've made the changes you suggested here and in issue [TW-2052].

File: content/en/docs/appstore/modules/mendix-sso.md [hunk 3]
```diff
@@ -38,15 +45,307 @@ You can use this module when you want your application's end-users to sign in wi
 ### 1.3 Limitations
 
 * Mendix SSO is not recommended for Production deployments as the Mendix SSO sign in screens are Mendix branded.
```
Comment (JaapF) [inline content/en/docs/appstore/modules/mendix-sso.md]: @MarkvanMents "not for production" makes it sound as if the module itself is inferior. Given the typical use case scenarios I suggest to say:

"Usage of Mendix SSO will present endusers with screens that are Mendix branded, which makes the module not suitable beyond the typical usage scenario indicated in porevious section."

Comment (MarkvanMents) [reply content/en/docs/appstore/modules/mendix-sso.md]: Done: https://github.com/mendix/docs/pull/5771/commits/e7be018dc68c29d52e053b4a7c968738c902b66d

**Comment by MarkvanMents** [comment]
Discussed over Slack. Merging.

**Pull request merged**